jobserver/urls.py

```diff
@@ -122,10 +121,11 @@
         WorkspaceReleaseList.as_view(),
         name="workspace-release-list",
     ),
+    path("releases/<pk>/", ReleaseDetail.as_view(), name="release-detail"),
     path(
-        "releases/<release>",
-        WorkspaceReleaseView.as_view(),
-        name="workspace-release",
+        "releases/<pk>/<path:path>",
```

Comment:
I'm confused by this, it seems to link directly to a file, rather than to the SPA pointing to a release?